Today/Controllers/ReminderDetailViewController/ReminderDetailViewController.swift

```diff
@@ -13,7 +13,7 @@ class ReminderDetailViewController: UITableViewController {
     
     // MARK: - Static Properties
     
-    static let reminderDetailCellIdentifier = "ReminderDetailCell"
+    static let reminderDetailCellIdentifier = Identifiers.reminderDetailCellIdentifier
```

Comment:
Not getting cell ids from extension ?

Reply:
No, was using hard coded string in this case.